### PR TITLE
Tweak ros2_overview tutorial.

### PR DESCRIPTION
**Pull request by caguero**
# 🦟 Bug fix

This patch fixes one typo and the figure (the parameter values were wrong). Thanks @azeey for catching it.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by azeey — COMMENTED** [review]
Thanks @caguero. One other thing I noticed on the image on the right is that it's not clear that Gazebo and the bridge are running in the same container and nav2 is running outside of that container. Could you add something to the graphics to group Gazebo and the bridge together?

**Review by ahcorde — APPROVED** [review]

**Comment by azeey** [comment]
I see. Should we have a separate diagram to show Gazebo and bridge in the same container and the consumer ROS node be a separate process? I think that's the most common use case.

**Comment by caguero** [comment]
> I see. Should we have a separate diagram to show Gazebo and bridge in the same container and the consumer ROS node be a separate process? I think that's the most common use case.

Thanks for the suggestions. Updated diagram and text added in [e83899c](https://github.com/gazebosim/docs/pull/531/commits/e83899c093825ef48a234712c2189abd2d3da328).

File: harmonic/ros2_overview.md [hunk 1]
```diff
@@ -57,12 +57,17 @@ ROS nodes will be intraprocess.
...
 the idea of not using composition. All the three example nodes are standalone
-nodes, and they can talk via interprocess communication. The center diagram
-represents the scenario where we use composition and we start our own container
-from our own launch file. All communication is intraprocess here. The right
-diagram is still using composition but the launch file doesn't start the
-container directly. This setup by itself will not work until you start an
-external ROS container (manually or via a separate launch file).
+nodes, and they can talk via interprocess communication using the bridge.
+The center diagram represents the scenario where we can use composition and our
+own ROS container between Gazebo and the bridge, but there's an additional
```
Comment (azeey) [inline harmonic/ros2_overview.md:62]: Reading this, it's not clear which perspective "own" refers to. That is, as a reader, I might think my own ROS container, but this is referring to `ros_gz`'s container.

```suggestion
The center diagram represents the scenario where we can use composition with a ROS container created by a `ros_gz` launch file containing both Gazebo and the bridge, and an additional
```

File: harmonic/ros2_overview.md [hunk 2]
```diff
@@ -57,12 +57,17 @@ ROS nodes will be intraprocess.
...
+consumer node outside that we cannot control. All communication between Gazebo
+and the bridge is intraprocess and interprocess between the external consumer
+node and the bridge.
+The diagram on the right side is using composition across all nodes but the
+launch file doesn't start our own container directly. This setup by itself will
```
Comment (azeey) [inline harmonic/ros2_overview.md:67]:
```suggestion
`ros_gz` launch file doesn't start its own container directly. This setup by itself will
```

**Review by azeey — APPROVED** [review]

**Pull request merged**